src/installation/installation.md

```diff
@@ -66,25 +66,29 @@ the helm value "kubed.enabled" to "false".
 Epinio is using an S3 compatible storage to store the application source code.
 This chart will install [Minio](https://min.io/) when `.Values.minio.enabled` is
 true (default). Any S3 compatible solution can be used instead by setting this
-value to `false` and using [the values under `s3`](https://github.com/epinio/helm-charts/blob/main/chart/epinio/values.yaml#L44)
-to point to the desired S3 server. 
+value to `false` and using [the values under `s3`](https://github.com/epinio/helm-charts/blob/b389a4875af9f03b484a911c49a14f834ba04b64/chart/epinio/values.yaml#L44)
```

Comment:
The link was previously set to a build ID, however I asked the team if we could get something more generic to avoid to change it too often. The "main" link was provided, so I would encourage to merge the build into main instead of changing the link here.

It's really just an opinion for a better docs maintenance.

Reply:
This is what GitHub gives you as a "permalink". The problem with "main" is that, as code changes, the link may point to completely irrelevant code. Permalinks are not perfect but at least they always refer to the same code which was relevant at the time of writing.